templates/template/maintenanceMode.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends "template/UI.html.twig" %}
+
+{% block title %}
+    Site is temporarily under maintenance
+{% endblock %}
+
+{% block body %}
+    <h1>Site is temporarily under maintenance</h1>
```

Comment:
The verbiage could use some work, and it needs the guy with the shovel image too. ;-)